src/seedu/addressbook/Main.java

```diff
@@ -104,14 +104,18 @@ private void recordResult(CommandResult result) {
      * @return result of the command
      */
     private CommandResult executeCommand(Command command)  {
+        if(!storage.exist()){
+        	ui.showToUser(storage.getWarning());
+        }
+        	
         try {
-            command.setData(addressBook, lastShownList);
-            CommandResult result = command.execute();
-            storage.save(addressBook);
-            return result;
+        	command.setData(addressBook, lastShownList);
```

Comment:
Indentation problem here?. Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.